modules/apps/27-interchain-accounts/types/packet_test.go

```diff
@@ -82,3 +85,70 @@ func (suite *TypesTestSuite) TestValidateBasic() {
 		})
 	}
 }
+
+func (suite *TypesTestSuite) TestPacketDataProvider() {
+	expCallbackAddr := ibctesting.TestAccAddress
+
+	testCases := []struct {
+		name              string
+		packetData        types.InterchainAccountPacketData
+		expAdditionalData map[string]interface{}
```

Comment:
Maybe `expCustomData` or `expMemoData`?
```suggestion
		expCustomData map[string]interface{}
```

Reply:
done